package io.craigmiller160.datamapper.exception;

public class MappingException {
}
